client/landing/gutenboarding/components/signup-form/index.tsx

```diff
@@ -79,7 +79,7 @@ const SignupForm = ( { onRequestClose }: Props ) => {
 	};
 
 	const tos = __experimentalCreateInterpolateElement(
-		NO__( 'By creating an account you agree to our <link_to_tos>Terms of Service</link_to_tos>.' ),
+		__( 'By creating an account you agree to our <link_to_tos>Terms of Service</link_to_tos>.' ),
 		{
 			link_to_tos: <ExternalLink href="https://wordpress.com/tos/" />,
```

Comment:
do we need to wrap this URL in some i18n function? E.g. `https://es.wordpress.com/tos/`

Reply:
04146792313c0d9e42fe43d1303fe32e5b400deb